client/src/catalog-publish.html

```diff
@@ -214,7 +214,13 @@ <h1 class="page-title">Publish your element</h1>
           <li>Attributes of your element and possible values</li>
           <li>A code sample showing how to include and use your element in an HTML page</li>
         </ul>
-        <p>For Polymer elements, see "<a href="https://www.polymer-project.org/1.0/docs/tools/documentation">Document your elements</a>" for more information.
+        <p>For Polymer elements, see "<a href="https://www.polymer-project.org/1.0/docs/tools/documentation">Document your elements</a>" for more information.</p>
+
+        <h2>How do I use data binding in an inline demo?</h2>
+        <p>Polymer supports a HTML data binding feature, which automatically performs data binding inside custom elements. To inform Polymer to perform data binding inside your inline demo, place your data-binding HTML in <code>&lt;template is="dom-bind"&gt;&lt;/template&gt;</code>. Ensure there's no script elements inside your template. If your snippet is simply HTML, you can do this in your hidden code:
```

Comment:
To inform...perform reads weird.

Maybe "To allow data binding at the top level ..."